pkg/model/resource/resource_test.go

```diff
@@ -311,9 +238,6 @@ var _ = Describe("Resource", func() {
 			Expect(resource.Package).To(Equal(path.Join("k8s.io", "api", options.Group, options.Version)))
 			Expect(resource.Domain).To(Equal("authentication.k8s.io"))
 
```

Comment:
Same minor NIT here: there is an extra blank line.

Reply:
solved